jwql/website/apps/jwql/data_containers.py

```diff
@@ -85,22 +87,20 @@
     setup()
 
     from .forms import MnemonicSearchForm, MnemonicQueryForm, MnemonicExplorationForm
-    from jwql.website.apps.jwql.models import Observation, Proposal, RootFileInfo
+    from jwql.website.apps.jwql.models import Observation, Proposal, RootFileInfo, Anomalies
     check_config_for_key('auth_mast')
     configs = get_config()
     auth_mast = configs['auth_mast']
     mast_flavour = '.'.join(auth_mast.split('.')[1:])
     from astropy import config
     conf = config.get_config('astroquery')
     conf['mast'] = {'server': 'https://{}'.format(mast_flavour)}
-
-__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
-if not ON_GITHUB_ACTIONS and not ON_READTHEDOCS:
     FILESYSTEM_DIR = configs['filesystem']
```

Comment:
I think the "if not ON_GITHUB_ACTIONS and not ON_READTHEDOCS:" is still required here, right?

Reply:
Those following lines of code get bumped up into the preceding group which has the same check in place.  I just merged all elements instead of having two groups with the same `if not ON_GITHUB_ACTIONS and not ON_READTHEDOCS:`